Source/Core/Core/HW/CPU.cpp

```diff
@@ -86,12 +89,34 @@ static void ExecutePendingJobs(std::unique_lock<std::mutex>& state_lock)
   }
 }
 
+static void RunTimer()
+{
+  const std::string game_id = SConfig::GetInstance().GetGameID();
+  TimePlayed time_played(game_id);
+  Common::Timer time_tracker;
+  time_tracker.Start();
+
+  while (s_state != State::PowerDown)
+  {
+    if (time_tracker.ElapsedMs() > 60000)
+    {
+      auto emulated_milliseconds = time_tracker.ElapsedMs(); 
+      time_tracker.Stop();
+      time_played.AddTime(emulated_milliseconds);
+      time_tracker.Start();
+    }
+  }
+}
```

Comment:
There needs to be some kind of event or sleep or *something* in here so this thread isn't just pegging a whole CPU core at all times.